src/kaleidoscope/plugin/IdleLEDs.cpp

```diff
@@ -25,23 +26,33 @@ uint16_t IdleLEDs::idle_time_limit = 600; // 10 minutes
 uint32_t IdleLEDs::end_time_;
 
 EventHandlerResult IdleLEDs::beforeEachCycle() {
-  if (!::LEDControl.paused &&
-      (Kaleidoscope.millisAtCycleStart() >= end_time_)) {
+  if (Kaleidoscope.millisAtCycleStart() < end_time_)
+    return EventHandlerResult::OK;
+
+  if (::Kaleidoscope.has_leds && !::LEDControl.paused) {
     ::LEDControl.set_all_leds_to(CRGB(0, 0, 0));
     ::LEDControl.syncLeds();
 
     ::LEDControl.paused = true;
   }
+  if (::UnderglowControl.has_leds() && !::UnderglowControl.isPaused()) {
+    ::UnderglowControl.setColor(0, 0, 0);
```

Comment:
I feel weird that this is "setColor(0,0,0)" and the LEDControl version is set_all_leds_to(CRGB(0,0,0)).

(LEDControl should at least grow setAllLedsTo() and this API and that one should use the same name and calling convention)

Reply:
Agreed. My thinking here was to play with names, to come up with something better than what we have in `LEDControl` (and then transition `LEDControl` over to the new name too). I'm not really fond of `setCrgbAt` for example, because it has the typename in the name too.

Btw, `setColor(CRGB(0, 0, 0))` also works, and so does `set_all_leds_to(r, g, b)`.